docs/AboutUs.md

```diff
@@ -48,12 +48,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Dev Ops + Threading
 
-### James Doe
+### Mohamed Hamas
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/hamas.png" width="200px">
```

Comment:
The profile image filename might wanna change to your github username too, for the dashboard to capture

Reply:
Rectified this too